x-pack/apm-server/sampling/processor_test.go

```diff
@@ -573,87 +573,6 @@ func TestStorageMonitoring(t *testing.T) {
 	assert.NotZero(t, metrics.Ints, "sampling.storage.value_log_size")
 }
 
-func TestStorageGC(t *testing.T) {
```

Comment:
For reviewers: this will fail because the meaning of vlog files changed, see https://github.com/dgraph-io/badger/pull/1555

Reply:
Thinking about this again, I'm not sure we should just remove the test. GC is still needed, it's just that how it works has changed. Can we instead update the test?